crates/virtio-queue/src/lib.rs

```diff
@@ -718,254 +731,20 @@ impl<M: GuestAddressSpace> Queue<M> {
     }
 }
 
-#[allow(missing_docs)]
-#[cfg(feature = "test-utils")]
-pub mod test_utils {
+#[cfg(test)]
+mod tests {
     use super::*;
+    use mock::*;
```

Comment:
It is a best practice to only import what you're using from the module. Right now you might be using everything from it, but as we add functionality to `mock` that will not be the case anymore. This is why it's better to be explicit in your imports.

Reply:
Done. Thank you for the tip.